From a5ba6e407c4658cb2c900f4cc0048984c79432aa Mon Sep 17 00:00:00 2001
From: Thomas BOURGOIN <thomas.bourgoin@st.com>
Date: Mon, 16 Jan 2023 10:04:34 +0100
Subject: [PATCH 395/529] ARM: dts: stm32: add CRC support for STM32MP13x SoC
 family

This patch adds the CRC support for STM32MP13x SoC family.

Signed-off-by: Thomas BOURGOIN <thomas.bourgoin@st.com>
Change-Id: I4296506f3951baad5ea08eaf4d2a9bf286e41708
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/285043
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm/boot/dts/stm32mp131.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm/boot/dts/stm32mp131.dtsi
+++ b/arch/arm/boot/dts/stm32mp131.dtsi
@@ -1469,6 +1469,13 @@
 			status = "disabled";
 		};
 
+		crc1: crc@58009000 {
+			compatible = "st,stm32f7-crc";
+			reg = <0x58009000 0x400>;
+			clocks = <&rcc CRC1>;
+			status = "disabled";
+		};
+
 		eth1: eth1@5800a000 {
 			compatible = "snps,dwmac-4.20a", "st,stm32mp13-dwmac";
 			reg = <0x5800a000 0x2000>;
